module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,41 @@
+#include "sort.hpp"
+
+#include <algorithm>
+#include <cctype>
+#include <deque>
+#include <iostream>
+
+std::deque<std::string> fillDequeWithValuesFromForwardList(std::forward_list<std::string>& listWithWords) {
+    std::deque<std::string> dequeWithWords{};
+    dequeWithWords.insert(dequeWithWords.begin(), listWithWords.begin(), listWithWords.end());
+
+    return dequeWithWords;
+}
+
+std::deque<std::string> changeLettersToLowercase(std::deque<std::string>& dequeOfWords) {
+    for (auto& word : dequeOfWords) {
+        std::transform(word.begin(), word.end(), word.begin(), [](auto& charackter) {
+            return std::tolower(charackter);
+        });
+    }
+    return dequeOfWords;
+}
+
+void eraseListWithWords(std::forward_list<std::string>& listWithWords) {
+    listWithWords.clear();
+}
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>& listWithWords) {
+    std::deque<std::string> dequeWithWords = fillDequeWithValuesFromForwardList(listWithWords);
+    changeLettersToLowercase(dequeWithWords);
+    eraseListWithWords(listWithWords);
```

Comment:
Why making a function for this? You could just call clear() here.

Reply:
I agree with that.